CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
+## v0.0.? - 20xx-yy-zz - implement full-fledged and default healthchecks
```

Comment:
Probably wouldn't hurt to go ahead and make the next one a minor, or maybe even major, release. 

```suggestion
## v0.1.0 - 20xx-yy-zz - implement full-fledged and default healthchecks
```

Reply:
Pushed that change.